code/go/0chain.net/smartcontract/minersc/models.go

```diff
@@ -659,10 +661,11 @@ func (mn *MinerNode) decodeFromValues(params url.Values) error {
 // nodeWithVCPoolLock represents a MinerNode that use ViewChangeLock as tokenLockInterface
 // it is for decoding MinerNode bytes
 type nodeWithVCPoolLock struct {
-	*SimpleNode `json:"simple_miner"`
-	Pending     map[string]*DelegatePoolWithVCPoolLock `json:"pending,omitempty"`
-	Active      map[string]*DelegatePoolWithVCPoolLock `json:"active,omitempty"`
-	Deleting    map[string]*DelegatePoolWithVCPoolLock `json:"deleting,omitempty"`
+	*SimpleNode         `json:"simple_miner"`
+	stakepool.StakePool `json:"stakepool.StakePool"`
```

Comment:
We'd better use `json:"stakepool"` instead of `json:"stakepool.StakePool"`.

Reply:
Thanks. These are changes from the next PR slipped in. There should be no changes to `minersc`.